README.md

```diff
@@ -144,6 +144,10 @@ versions, which this module supports. It's recommended that you set the
 you have installed. By default the module will configure for the latest
 version available.
 
+# Soft Dependencies
+
+If you use OS which runs on systemd you may consider to download `camptocamp/systemd` module too for compatibility.
```

Comment:
I think it should be mentioned that this only a dependency when using the puppetserver features, not for agent-only.

Reply:
Added it to the description just now :).